crispy_forms/tests/results/bootstrap4/test_form_helper/bootstrap_form_show_errors_bs4_false.html

```diff
@@ -3,7 +3,7 @@
         <label for="id_email" class=" requiredField"> email<span class="asteriskField">*</span> </label>
         <div class="">
             <div class="input-group">
-                <input type="text" name="email" value="invalidemail" maxlength="30" class="textinput textInput inputtext form-control is-invalid form-control is-invalid" required id="id_email" />
+                <input type="text" name="email" value="invalidemail" maxlength="30" class="textinput textInput inputtext form-control is-invalid is-invalid" required id="id_email" />
```

Comment:
`is-invalid` is still in `crispy_forms_field.py` and will be removed as a separate change.